samples/subsys/ipc/rpmsg_demo/prj.conf

```diff
@@ -0,0 +1,15 @@
+# enable c++ support
+CONFIG_NEWLIB_LIBC=y # use newlib over minimal
+CONFIG_CPLUSPLUS=y
+CONFIG_LIB_CPLUSPLUS=y
+CONFIG_STD_CPP20=y
```

Comment:
No, it is not necessary. We can change it to C. No C++ std features are anyway used.

Reply:
As @arnopo suggested, it would be nice to modify the existing [`openamp_rsc_table` sample](https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/subsys/ipc/openamp_rsc_table) to work on the i.MX8MM platform.